src/zui/l10n/messageIds.ts

```diff
@@ -24,6 +24,20 @@ export default makeMessages('zui', {
     copiedValue: m<{ value: string }>('Copied "{value}"'),
     copy: m('Copy'),
   },
+  createPerson: {
+    cancel: m('Cancel'),
+    createBtn: m('Create person'),
+    genders: {
+      f: m('Female'),
+      m: m('Male'),
+      o: m('Other'),
+      unknown: m('Unknown'),
+    },
+    showAllFields: m('Show all fields'),
+    title: m('Create person'),
+    url: m('URL'),
+    validationWarning: m<{ field: string }>('Please add a valid {field}'),
```

Comment:
I don't think that it will be possible to translate this string, because it relies on all values for `{field}` having the same grammatical gender.